src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/QueryExecutor.cs

```diff
@@ -651,10 +644,7 @@ protected virtual void WriteEvent(DbCommand command, IPersistentRepresentation e
             }
             else
             {
-                if (serializer.IncludeManifest)
```

Comment:
Why did you remove this ?

Reply:
If this is set to false for any serializer, we can't `ReadEvent` because there's no type name in the `manifest` field and all the specs fail.  Not sure of another way around it. https://github.com/akkadotnet/akka.net/pull/3052/files/702218b1cb29718e86cb7d6c1547cc61217c900d#diff-cb3053db11018a282911ee665d86db6eR686